static/src/javascripts/projects/common/modules/accessibility/main.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+import userPrefs from 'common/modules/user-prefs';
+
+const KEY_PREFIX = 'accessibility';
+
+const saveState = (state: Object): void => {
+    Object.keys(state).forEach(key => {
+        userPrefs.set(`${KEY_PREFIX}.${key}`, state[key]);
+    });
+};
+
+const getStoredValue = (key: string): boolean =>
+    userPrefs.get(`${KEY_PREFIX}.${key}`) !== false;
+
+const isOn = (key: string): boolean => getStoredValue(key) === true;
+
+export { KEY_PREFIX, saveState, isOn };
```

Comment:
Why do we export KEY_PREFIX? It doesn't appear to be used anywhere outside of this module? It seems like a  strange pattern too, changing the value of KEY_PREFIX would lead to am API change of the module.

Reply:
It's used in the test spec, I guess we're exporting it to keep things DRY. I think exporting it is a bit overkill TBH, it doesn't make sense from this module's perspective. I'd prefer to see it hardcoded into the tests.

Reply:
Ah I didn't realise there's a test spec for this module! I guess this should be converted as part of this PR then? I agree with you're point about exporting this property it being overkill.

Reply:
Ok, chatted with @SiAdcock  about whether a module's spec should be converted as part of the conversion process and agreed it's not necessary at this point.